### PR TITLE
Follow up on #169

### PR DESCRIPTION
**Pull request by iitzco-asapp**
See #169

### CODE REVIEW

File: flambe/runner/run.py [hunk 1]
```diff
@@ -114,6 +119,8 @@ def main(args: argparse.Namespace) -> None:
                              'when using this flag as it could have undesired effects.')
     parser.add_argument('-s', '--secrets',
                         type=str, default=os.path.join(FLAMBE_GLOBAL_FOLDER, "secrets.ini"))
+    parser.add_argument('-d', '--debug', action='store_true',
+                        help='Enable debug mode, overriding config.yaml setting')
```
Comment (nmatthews-asapp) [inline flambe/runner/run.py]: This no longer overrides `config.yaml` setting, right? Also I think we should explain a bit more what debug does, e.g. explicitly say "For an experiment job, Ray will run in a single thread if debug mode is enabled."

Comment (iitzco-asapp) [reply flambe/runner/run.py]: Done.

File: flambe/runner/run.py [hunk 2]
```diff
@@ -38,6 +38,10 @@ def main(args: argparse.Namespace) -> None:
         print(cl.RA(ASCII_LOGO))
         print(cl.BL(f"VERSION: {flambe.__version__}\n"))
 
+    if args.debug and args.cluster is not None:
```
Comment (jeremyasapp) [inline flambe/runner/run.py]: Are we sure that we want to deactivate all of the `debug` functionalities in remote execution?

Comment (iitzco-asapp) [reply flambe/runner/run.py]: It's not clear to me how `debug` can work in a remote execution. Until we figure that out, I suggest `debug` just works locally for any runnable.

Comment (jeremyasapp) [reply flambe/runner/run.py]: sounds good

**Pull request merged**